### DISCUSSION TITLE
can't merge

### DISCUSSION THREAD

**Discussion by keding-com** [community/community#33148] [Other Feature Feedback, Questions, & Ideas]
I'm the manager but can't merge.
always loading.
![螢幕擷取畫面 2022-09-15 161703](https://user-images.githubusercontent.com/46552797/190352476-075b5def-6df0-435e-89d4-7eb902cea9c6.png)

**Comment by jfullstackdev** [comment]
you might be referring to the parent repo of this fork ? 

![image](https://user-images.githubusercontent.com/47092464/190387864-a5e3977d-4eb8-4016-9395-8c6d01c6d521.png)

if you don't have access to the parent repo, then you can't commit, you can only change your fork
